### DISCUSSION TITLE
i use x86 macos with zigbuild buid aarch64-apple-darwin , but strip will cause exe error

### DISCUSSION THREAD

**Discussion by i18nsite** [rust-cross/cargo-zigbuild#222] [Q&A]
<img width="424" alt="image" src="https://github.com/rust-cross/cargo-zigbuild/assets/145643935/74d367e8-be76-4b0e-8200-932571a702cd">

i use x86 macos with zigbuild buid aarch64-apple-darwin , but strip will cause exe error

i use mac m2 strip , strip this file again , the file can run 
```
❯ file i18n
i18n: Mach-O 64-bit executable arm64
❯ ./i18n
zsh: killed     ./i18n
❯ ls -alh i18n
Permissions Size User Date Modified Name
.rwxr-xr-x  531k z     2 2  00:15   i18n
❯ strip i18n
❯ ls -alh i18n
Permissions Size User Date Modified Name
.rwxr-xr-x  534k z     2 2  00:38   i18n
❯ ./i18n
aarch64-apple-darwin
```

I also try use aarch64-apple-darwin-cross strip , but it not works too
❯ docker run --platform linux/amd64 -it -v .:/app --rm i18nsite/aarch64-apple-darwin-cross /opt/osxcross/bin/aarch64-apple-darwin21.4-strip /app/i18n
/opt/osxcross/bin/aarch64-apple-darwin21.4-strip: warning: changes being made to the file will invalidate the code signature in: /app/i18n

**Comment by messense** [comment]
Then don't strip it.
